docs/client-sdk/javascript/tutorials/label-conversations.md

```diff
@@ -5,13 +5,19 @@ sidebar_position: 4
 
 # Use labels to differentiate multiple conversations between a pair of addresses
 
-With XMTP, a pair of blockchain account addresses can have multiple ongoing conversations. In addition, with XMTP, an app can display all conversations a user has, regardless of the app they used to create the conversation. This concept is known as a portable, or universal, inbox.
+With XMTP, a pair of blockchain account addresses can have multiple ongoing conversations. In addition, with XMTP, an app can display all conversations a user has, regardless of the app they used to create the conversation. This concept is known as an interoperable inbox.
 
 Alternatively, an app can implement conversation IDs and metadata to [filter conversations](filter-conversations). For example, a developer can choose to assign app-specific conversation IDs and metadata to filter and display only conversations created by their app.
 
-Apps that assign conversation IDs in this way have an effect on apps that use a portable inbox model. For example, a user might see multiple separate and ongoing conversations they are having with the same address, but originating in different apps.
+Apps that assign conversation IDs in this way have an effect on apps that use an interoperable inbox model. For example, a user might see multiple separate and ongoing conversations they are having with the same address, but originating from contexts with different conversation IDs. For example, the conversations may have originated in different apps or may have different subject lines.
 
-If you're using a portable inbox model for your app, you can display conversation labels based on conversation IDs and metadata to help clarify the user experience.
+If you're using an interoperable inbox model in your app, you can display conversation labels based on conversation IDs and metadata to help clarify the user experience.
+
+:::tip
+
+When providing an interoperable inbox, display only one conversation per address pair and conversation ID combination. This ensures that the two addresses exchange messages using the same conversation topic.
+
+:::
 
 To illustrate this scenario, let's take a look at a couple of conversations between `amal.eth` and `bola.eth`.
```

Comment:
Thoughts on illustrating via a real life scenario? E.g. Amal & Bola are using 'Lens' conversationId to chat about stuff across Lenverse and using a different 'OpenSea' conversationId to chat about an NFT transaction.

Different apps -> different converastionIds is an implication I think we want to move away from. It's something apps _can_ do but we don't want to imply that it's something they _should_.

Reply:
@yash-luna Thank you for this! I am getting it. I've updated the examples to reflect the scenarios you've shared. When you have a chance, let me know what you think!

https://junk-range-possible-kbgpesg1f-xmtp-labs.vercel.app/docs/client-sdk/javascript/tutorials/label-conversations

Once we have an example app using metadata to provide a label, we can update the code sample. Or perhaps Nick or Bhavya might have some thoughts on how to do it without needing an implementation first.